app/services/complex_cases/importer.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ComplexCases
+  class Importer
```

Comment:
the name `importer`, make me think that we are importing data from an external source,
Actually we could store the "COMPLEX_CASES" in a yaml, as we did with `/data/supplier_locations.yml`

Reply:
This follows the same pattern used in other reference data endpoints e.g. https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/services/identifier_types/importer.rb

I don't see any advantage in adding another abstraction layer by moving the hash to a yaml file at this point. If the data changes in future we can revisit this and think about soft deletions and other complexities, but it's not needed yet.

Reply:
I don't think it's good to have data hardcoded inside classes, but that's not an issue

Reply:
It would be hardcoded in a YAML file too, and would require a redeployment to update the contents regardless. I'm trying to keep this simple and then only add complexity later when the requirements dictate it (e.g. this data starts changing).